### ISSUE TITLE
Make unrequired field during on serialization

### ISSUE THREAD

**Issue by rayandu924** [marshmallow-code/marshmallow#2330]
In the load function, can we add the opposite of the partial argument to make fields required ?

**Comment by lafrech** [comment]
No. `partial` makes all fields not required. There is no equivalent argument to make all or some fields required at load time.

**Comment by rayandu924** [comment]
Could this feature be considered for a future release?

**Comment by sloria** [comment]
i don't think the use case warrants added API. you can declare a separate schema with the fields that you need to be required
